crates/driver/src/extract_defrefs.rs

```diff
@@ -447,7 +453,7 @@ pub async fn extract_defrefs(
     let sha_of_conf: Sha256Value = merged_config_str.as_bytes().into();
     let sha_of_conf_config = Arc::new(format!("{}", sha_of_conf));
 
-    let extractors = load_extractors(extract)?;
+    let extractors = load_extractors(extract).await?;
 
     let sha_to_extract_root = Box::leak(Box::new(opt.cache_path.join("sha_to_extract")));
```

Comment:
leak sounds scary... I better look that up...

Reply:
Leak is fun, its not `unsafe` since it tells rust to take an allocated on the heap thing and just forget about cleaning it up. So then we can guarantee (without doing something unsafe), that it will survive for the length of the application. So we can get a static reference to it. Its quite handy for config/command line apps, rather than having to use arc's/ or caring about reference lifetimes we can just pass it around as an immutable static value